package.json

```diff
@@ -586,15 +623,25 @@
                     "when": "view == azureLogicAppsExplorer"
                 },
                 {
-                    "command": "azureLogicApps.createLogicApp",
+                    "command": "azureLogicApps.createProject",
```

Comment:
Should Create Project and Add Build Def to Project be next to each other? Might make sense to group them together, but I also see this as a larger to smaller order, so maybe it's fine.

Reply:
The order is consistent with the way Azure Functions presents these options.